lib/install/config/webpacker.yml

```diff
@@ -33,7 +33,7 @@ development:
     host: localhost
     port: 8080
     https: false
-    hot_reloading: false
+    hmr: false
```

Comment:
@gauravtiwari @dhh FWIW, the [docs](https://webpack.js.org/configuration/dev-server/#devserver-hot) for `devServer.hot` use `hot` and not `hmr`.